drizztsaga/tpa/items_creatures.tpa

```diff
@@ -0,0 +1,235 @@
+// ---------------------------------------------------------------------------------------------------
+DEFINE_ACTION_MACRO ~DS_ITEMS_NERFING~ BEGIN
+// ---------------------------------------------------------------------------------------------------
+
+// +5 weapons into +3
+ACTION_FOR_EACH ~file~ IN
+				~F_ADAGGR~ // Snake Soul +5
+				~F_ICEAX5~ // Ice Slasher +5
+				~F_SW2H17~ // Belhifet's Sword of Grief +5
+BEGIN
+  COPY_EXISTING ~%file%.itm~ ~override~ // nerfing +5 weapon into "only" +3 weapon
+	WRITE_SHORT 0x60 3 // Enchantment Level
+    LPF ALTER_ITEM_HEADER INT_VAR type = 1 thac0_bonus = 3 damage_bonus = 3 END
+	PATCH_FOR_EACH offset IN 0xc 0x54 BEGIN
+	  READ_LONG ~%offset%~ desc_strref
+	  PATCH_IF (desc_strref >= 0 && desc_strref < 2147483646) BEGIN
+		READ_STRREF ~%offset%~ desc
+		INNER_PATCH_SAVE desc ~%desc%~ BEGIN
+		  REPLACE_TEXTUALLY CASE_INSENSITIVE EXACT_MATCH ~+ 5~ ~+ 3~ // change tach0 and damage modifier in name and description to +2
+		  REPLACE_TEXTUALLY CASE_INSENSITIVE EXACT_MATCH ~+ 5~ ~+ 3~ // change tach0 and damage modifier in name and description to +2
+		END
+	  END
+	  SAY_EVALUATED offset ~%desc%~
+	END
+  BUT_ONLY_IF_IT_CHANGES
+END
+
+// +4 weapons into +2
+ACTION_FOR_EACH ~file~ IN
+				~F_ARTSWD~ // Snake Bite +4
+				~F_BLUN35~ // Ice Star +4
+				~F_JARLAX~ // Battleaxe of Dispelling Fire +4
+				~F_SW2H09~ // Warblade +4
+BEGIN
+  COPY_EXISTING ~%file%.itm~ ~override~ // nerfing +4 weapon into "only" +2 weapon
+	WRITE_SHORT 0x60 2 // Enchantment Level
+    LPF ALTER_ITEM_HEADER INT_VAR type = 1 thac0_bonus = 2 damage_bonus = 2 END
+	PATCH_FOR_EACH offset IN 0xc 0x54 BEGIN
+	  READ_LONG ~%offset%~ desc_strref
+	  PATCH_IF (desc_strref >= 0 && desc_strref < 2147483646) BEGIN
+		READ_STRREF ~%offset%~ desc
+		INNER_PATCH_SAVE desc ~%desc%~ BEGIN
+		  REPLACE_TEXTUALLY CASE_INSENSITIVE EXACT_MATCH ~+4~ ~+2~ // change tach0 and damage modifier in name and description to +2
+		END
+	  END
+	  SAY_EVALUATED offset ~%desc%~
+	END
+  BUT_ONLY_IF_IT_CHANGES
+END
+
+// some +3 weapons into +2
+ACTION_FOR_EACH ~file~ IN
+				~F_XBOW08~ // Crossbow +3
+				~F_SCIM03~ // Scimitar +3
+BEGIN
+  COPY_EXISTING ~%file%.itm~ ~override~ // nerfing +3 weapon into "only" +2 weapon
+	WRITE_SHORT 0x60 2 // Enchantment Level
+    LPF ALTER_ITEM_HEADER INT_VAR type = 1 thac0_bonus = 2 damage_bonus = 2 END
+    LPF ALTER_ITEM_HEADER INT_VAR type = 4 thac0_bonus = 3 damage_bonus = 4 END
+	PATCH_FOR_EACH offset IN 0xc 0x54 BEGIN
+	  READ_LONG ~%offset%~ desc_strref
+	  PATCH_IF (desc_strref >= 0 && desc_strref < 2147483646) BEGIN
+		READ_STRREF ~%offset%~ desc
+		INNER_PATCH_SAVE desc ~%desc%~ BEGIN
+		  REPLACE_TEXTUALLY CASE_INSENSITIVE EXACT_MATCH ~+3~ ~+2~ // change tach0 and damage modifier in name and description to +2
+		  REPLACE_TEXTUALLY CASE_INSENSITIVE EXACT_MATCH ~+4~ ~+3~ // change tach0 modifier in name and description to +3
```

Comment:
You have changed only +3 items, so why +4 and +5 modification here as well?

Reply:
for the xbow, bonus for dmg are not the same other regular weapons. 
heavy xbow : +2 dmg

but you are right, the +4/+2 is useless.